internal/servers/worker/testing.go

```diff
@@ -207,6 +211,10 @@ func NewTestWorker(t *testing.T, opts *TestWorkerOpts) *TestWorker {
 		opts.Config.Worker.Name = opts.Name
 	}
 
+	if opts.StatusGracePeriodDuration > 0 {
```

Comment:
Is this needed?  Doesn't `SetStatusGracePeriodDuration` below handle both the zero and non-zero case?

Reply:
Yeah, it's one or the other. Probably a relic of maintenance, will fix either way.